content/scout/scores.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Docker Scout health scores
+description: |
+  Docker Scout health scores provide a supply chain assessment for Docker Hub
+  images, grading them from A to F based on various security policies.
+keywords: scout, health scores, evaluation, checks, grades, docker hub
+sitemap: false
+---
+
+> **Early Access**
+>
+> Health scores is an [Early Access](/release-lifecycle/#early-access-ea)
+> feature of Docker Scout. The feature is only available to organizations
+> participating in the early access program for this feature.
+{ .restricted }
+
+Docker Scout health scores provide a security assessment, and overall supply
+chain health, of images on Docker Hub, helping you determine whether an image
+meets your security standards. The scores range from A to F, where A represents
```

Comment:
instead of "meets your security standards", I'd change this to "meets foundational software supply chain security standards" or "meets established security best practices".

Reply:
+1, our long-term goal is to promote the OOTB policies as best practices so Tazin's copy will bring us closer to that point